src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml

```diff
@@ -27,6 +27,18 @@
                                               x:Name="Background_analysis_scope_full_solution"
                                               Content="{x:Static local:AdvancedOptionPageStrings.Option_Background_Analysis_Scope_Full_Solution}"/>
                     </StackPanel>
+                    <StackPanel>
+                        <CheckBox x:Name="DisplayDiagnosticsInline"
+                              Content="{x:Static local:AdvancedOptionPageStrings.Option_Display_diagnostics_inline_experimental}"/>
+                        <StackPanel Margin="15, 0, 0, 0">
+                            <RadioButton GroupName="Inline_diagnostic_location"
+                                     x:Name="place_at_end_of_line_of_code"
```

Comment:
the name should match the option.  so at_the_end_of_the_line_of_code

Reply:
basically, we can have unique english strings we show users.  we shouldn't then have random 'sorta similar' strings for those same strings internally for how we refernece those.

Reply:
Gotcha,  changed